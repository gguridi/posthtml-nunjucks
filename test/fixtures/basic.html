<html>
    <h1>{{ name }}</h1>
    <ul>
        {% for name, item in items -%}
        <li>{{ name }}: {{ item }}</li>
        {% endfor -%}
    </ul>
</html>
