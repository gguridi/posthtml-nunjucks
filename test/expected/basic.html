<html>
    <h1>test-name</h1>
    <ul>
        <li>keyA: valueA</li>
        <li>keyB: valueB</li>
    </ul>
</html>
